e2e/service_cosmos_test.go

```diff
@@ -3,51 +3,162 @@
 package e2e
 
 import (
+	"encoding/json"
+	"fmt"
+	"maps"
 	"net/http"
+	"strings"
 
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 	vegeta "github.com/tsenart/vegeta/lib"
 
 	"github.com/buildwithgrove/path/qos/jsonrpc"
 )
 
-// TODO_NEXT(@Commoddity): add test cases for XRPL-EVM Testnet and Pocket Shannon once
-// suppliers for both are updated to latest Relay Miner version that supports the new RPC types.
-// TODO_NEXT(@Commoddity): add tests for Cosmos SDK & JSON-RPC types once suppliers are updated.
+func getRPCTypeFromString(rpcType string) sharedtypes.RPCType {
```

Comment:
Please add godoc

Reply:
Can we move this function outside of `e2e/service_cosmos_test.go` into `e2e/???` because it's not cosmos specific?

Reply:
Done.